src/ray/raylet/scheduling/cluster_resource_scheduler.cc

```diff
@@ -592,6 +592,13 @@ std::string ClusterResourceScheduler::DebugString(void) const {
   return buffer.str();
 }
 
+uint64_t ClusterResourceScheduler::GetNumCpus(void) const {
```

Comment:
Could it be possible that the user has num_cpus=2.5 ?

Reply:
Btw, why void in the parameter?

Reply:
> Could it be possible that the user has num_cpus=2.5 will this lead to the precision issue?
I believe resource _totals_ are required to be whole numbers. 

> Btw, why void in the parameter?
Idk, removing.